Packs/Core/Playbooks/playbook-A_source_host_created_uncommon_remote_scheduled_task.yml

```diff
@@ -0,0 +1,1344 @@
+id: A source host created uncommon remote scheduled task
+version: -1
+contentitemexportablefields:
+  contentitemfields: {}
+name: A source host created uncommon remote scheduled task
+description: |-
+  This playbook handles "Uncommon remote scheduled task creation" alerts, which are generated on the source host that created the remote scheduled task.
+
+  Playbook Stages:
+
+  Analysis:
+
+  - The playbook verifies whether the causality process is signed. If the process is not signed, it proceeds with remediation actions; otherwise, it continues investigating the alert.
+
+  Investigation:
+  
+  During the alert investigation, the playbook will perform the following:
+
+  - Searches for related Cortex XSIAM alerts on the endpoint that use the following MITRE techniques to identify malicious activity: T1202 - Indirect Command Execution, T1021 - Remote Services.
+  - Searches for related Cortex XSIAM agent alerts on the remote endpoint, to determine if the creation of the scheduled task is part of an attack pattern.
+  - Searches for suspicious command-line parameters indicating a malicious scheduled task.
+
+  Remediation:
+
+  - Automatically disable the malicious scheduled task on the remote host.
+  - Terminate the causality process.
+  - Automatically close the alert.
+tags:
+- TA0002 - Execution
+- T1053 - Scheduled Task/Job
+starttaskid: "0"
+tasks:
+  "0":
+    id: "0"
+    taskid: e27de70b-ada6-422e-81fe-6950a566b050
+    type: start
+    task:
+      id: e27de70b-ada6-422e-81fe-6950a566b050
+      version: -1
+      name: ""
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "27"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 410,
+          "y": -390
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "1":
+    id: "1"
+    taskid: 6f7359e7-6ace-48a6-8f72-c30dc8bce825
+    type: title
+    task:
+      id: 6f7359e7-6ace-48a6-8f72-c30dc8bce825
+      version: -1
+      name: Investigation
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "47"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 760,
+          "y": 540
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "2":
+    id: "2"
+    taskid: 3ddedabb-1395-4234-81f5-a3b098a10721
+    type: condition
+    task:
+      id: 3ddedabb-1395-4234-81f5-a3b098a10721
+      version: -1
+      name: Is the causality process signed and prevalent?
```

Comment:
in the condition you are checking the opposite - if the process is not signed and not prevalent

Reply:
Done.